tests/unit/exception_logging_test.cc

```diff
@@ -177,6 +177,7 @@ BOOST_AUTO_TEST_CASE(unknown_object_thrown_test) {
 
 }
 
+#if FMT_VERSION < 80000
```

Comment:
Should have a comment here explaining why this is not needed with new fmt.

Reply:
sure. added.